mypy.ini

```diff
@@ -0,0 +1,1070 @@
+[mypy]
+python_version = 3.10
+pretty = false
+exclude = (?x)(
+          docs|
+          examples
+          )
+
+enable_error_code = ignore-without-code
```

Comment:
```suggestion
enable_error_code = ignore-without-code

plugins = numpy.typing.mypy_plugin
```

Reply:
I think we will use this plugin frequently.

Reply:
https://pydoc.dev/numpy/latest/numpy.typing.mypy_plugin.html

Reply:
Making a note to add this to PlasmaPy!